lnbits/core/services.py

```diff
@@ -371,10 +375,12 @@ class PaymentKwargs(TypedDict):
                 logger.debug(f"payment successful {payment.checking_id}")
         elif payment.checking_id is None and payment.ok is False:
             # payment failed
-            logger.warning("backend sent payment failure")
             async with db.connect() as conn:
-                logger.debug(f"deleting temporary payment {temp_id}")
-                await delete_wallet_payment(temp_id, wallet_id, conn=conn)
+                await update_payment_status(
```

Comment:
a `TRACE` level logger might come in handy

Reply:
added debug